import_map.json

```diff
@@ -3,14 +3,11 @@
     "$fresh/": "https://deno.land/x/fresh@1.2.0/",
     "preact": "https://esm.sh/preact@10.16.0",
     "preact/": "https://esm.sh/preact@10.16.0/",
-    "react": "https://esm.sh/@preact/legacy-compat?external=preact",
-    "react-dom": "https://esm.sh/@preact/legacy-compat?external=preact",
-    "create-react-class": "https://esm.sh/@preact/legacy-compat?external=preact",
```

Comment:
We don't need the base dependency on `react` anymore?

Reply:
Not any more under my understanding, since the `legacy-compat` was to make `react-json-inspector` library to work. We can keep it aliasing to `preact/compat` instead I think, but for now it will be only used on `react-json-view-lite`, so I think it is fine to keep the aliasing in the new library import line.